### PR TITLE
[Feat] Searching ExFeeder in text will show ExFeeder nominees

### PR DESCRIPTION
**Pull request by Rizvida1**
#### Changes introduced

Searching ExFeeder in search bar will show exfeeder nominees

#### Related issue(s)

- Related to #1123

### CODE REVIEW

File: services/backend/src/core/search/util/getAllProfiles.js [hunk 1]
```diff
@@ -398,6 +398,10 @@ async function getAllUsers(language, userId, request) {
         : undefined;
     }
 
+    if (info.exFeeder) {
+      info.exFeederText = "ExFeeder Ex-Feeder";
+    }
+
```
Comment (dynamic11) [inline services/backend/src/core/search/util/getAllProfiles.js]: we also need the french. according to mena it has another term. please confirm with her

**Review by dynamic11 — CHANGES_REQUESTED** [review]
please see my comment

File: services/backend/src/core/search/util/fuzzySearch.js [hunk 2]
```diff
@@ -38,6 +38,7 @@ async function fuzzySearch(profiles, searchValue) {
       "teams",
       "telephone",
       "tenure",
+      "exFeederText",
```
Comment (dynamic11) [inline services/backend/src/core/search/util/fuzzySearch.js:41]: you will also have to display this match in the fuzzy search match list on the result card

File: services/backend/package-lock.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (dynamic11) [inline services/backend/package-lock.json]: why is this being added?

**Comment by dynamic11** [comment]
can you also add a test to fuzzy search for this? because it is a corner case i want to make sure we have a test for it so we don't break it in t future.

**Review by dynamic11 — APPROVED** [review]
